tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true,
+    "skipLibCheck": true,
+    "lib": ["es2022"],
+    "target": "es2022",
+    "module": "Node16",
+    "moduleResolution": "node16",
+
+    "moduleDetection": "force",
+    "isolatedModules": true,
+    "verbatimModuleSyntax": true,
+
+    "strict": true,
```

Comment:
I decided to enable strict checks to prevent us from unintentionally introduce `any` type, e.g. by forgetting to add type annotations to functions. We can revert this if it becomes too cumbersome.